### PR TITLE
adding mail to config

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## JIRA
https://issues.redhat.com/browse/DEL-2

## Changes
Add mailTo parameter to the job definitions for 3scale, fuse-online, amq-online

## Verification
No steps required - verified as described below.
The pipeline has been run here with a parameter override for my mail: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/3scale/job/lfdiscovery/44/parameters/
The email was recieved to my mail only rather than the delorean mail list..
Screenshot 2020-02-21 at 15 56 28

### CODE REVIEW

File: jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile [hunk 1]
```diff
@@ -131,10 +132,9 @@ pipeline {
         always {
             script {
                 final Map msg = readJSON text: CI_MESSAGE
-                String mailTo = 'cloud-services-delorean@redhat.com'
                 mail(
                         subject: "New build of ${msg.info.name} created in Brew",
-                        to: mailTo,
+                        to: "${mailTo}",
```
Comment (mikenairn) [inline jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: Is this change needed?

Comment (laurafitzgerald) [reply jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: No. Thought it had failed on that earlier. But tested it again and it works without it. Successful build here: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/3scale/job/lfdiscovery/51/console
